front/lib/dust_api.ts

```diff
@@ -215,7 +215,8 @@ export const DustAPI = {
   async createRunStream(
     projectId: string,
     dustUserId: string,
-    payload: DustAPICreateRunPayload
+    payload: DustAPICreateRunPayload,
+    onDustRunId?: (runId: string) => void
```

Comment:
This is the best option I found. LMK what you think @spolu !

Reply:
:+1: